design/ceph/ceph-csi-driver.md

```diff
@@ -138,7 +139,11 @@ automatically create storage classes based on that additional metadata.
 
 #### Select Flex Provisioning or CSI based on CephCluster CRD
 
-Currently the code requires changing numerous parameters to enable CSI.  This
+Currently the code requires changing numerous parameters to enable CSI. This
 document aims to change that to a single parameter. In the future it may be
 desirable to make this more of a "runtime" parameter that could be managed in
 the cluster CRD.
+
+### Support RBD Mirroring
+
+Start a Volume Replication sidecar container in the RBD Provisioner pod to start supporting RBD Mirroring between Ceph clusters.
```

Comment:
more details need to be added here on how to do failover and failback for DR. refer https://github.com/kube-storage/volume-replication-operator#design